### ISSUE TITLE
Error while installing AutoITLibrary 1.2.4 version.

### ISSUE THREAD

**Issue by acn-nithya** [nokia/robotframework-autoitlibrary#18]
**Error while trying to install AutoITLibrary version 1.2.4.   Below is the error in cmd. Could you please help me here?**

C:\Temp\robotframework-autoitlibrary-1.2.4>python setup.py install
Don't think we need to unregister the old one...
%SYSTEMROOT%\system32\regsvr32.exe /S C:\Program Files\Python35\Lib\site-package
s\AutoItLibrary\lib\AutoItX3.dll
Traceback (most recent call last):
  File "setup.py", line 75, in <module>
    subprocess.check_call(cmd, shell=True)
  File "C:\Program Files\Python35\lib\subprocess.py", line 271, in check_call
    raise CalledProcessError(retcode, cmd)
subprocess.CalledProcessError: Command '%SYSTEMROOT%\system32\regsvr32.exe /S C:
\Program Files\Python35\Lib\site-packages\AutoItLibrary\lib\AutoItX3.dll' return
ed non-zero exit status 3

C:\Temp\robotframework-autoitlibrary-1.2.4>

**I also tried installing using PIP command but again the below error shows up.**

C:\Temp\robotframework-autoitlibrary-1.2.4>pip install robotframework-autoitlibr
ary
Collecting robotframework-autoitlibrary
  Downloading  .../robotframework-autoitlibrary-1.2.4.zip (701kB)
    100% |################################| 706kB 17.6MB/s
Requirement already satisfied: pywin32 in c:\program files\python35\lib\site-pac
kages (from robotframework-autoitlibrary)
Requirement already satisfied: pillow in c:\program files\python35\lib\site-pack
ages (from robotframework-autoitlibrary)
Installing collected packages: robotframework-autoitlibrary
  Running setup.py install for robotframework-autoitlibrary ... error
    Complete output from command "c:\program files\python35\python.exe" -u -c "i
mport setuptools, tokenize;__file__='C:\\Users\\BA05476\\AppData\\Local\\Temp\\p
ip-build-t8iw_px1\\robotframework-autoitlibrary\\setup.py';f=getattr(tokenize, '
open', open)(__file__);code=f.read().replace('\r\n', '\n');f.close();exec(compil
e(code, __file__, 'exec'))" install --record C:\Users\BA05476\AppData\Local\Temp
\pip-mb2nou3f-record\install-record.txt --single-version-externally-managed --co
mpile:
    Don't think we need to unregister the old one...
    %SYSTEMROOT%\system32\regsvr32.exe /S c:\program files\python35\Lib\site-pac
kages\AutoItLibrary\lib\AutoItX3.dll
    Traceback (most recent call last):
      File "<string>", line 1, in <module>
      File "C:\Users\BA05476\AppData\Local\Temp\pip-build-t8iw_px1\robotframewor
k-autoitlibrary\setup.py", line 75, in <module>
        subprocess.check_call(cmd, shell=True)
      File "c:\program files\python35\lib\subprocess.py", line 271, in check_cal
l
        raise CalledProcessError(retcode, cmd)
    subprocess.CalledProcessError: Command '%SYSTEMROOT%\system32\regsvr32.exe /
S c:\program files\python35\Lib\site-packages\AutoItLibrary\lib\AutoItX3.dll' re
turned non-zero exit status 3

    ----------------------------------------
Command ""c:\program files\python35\python.exe" -u -c "import setuptools, tokeni
ze;__file__='C:\\Users\\BA05476\\AppData\\Local\\Temp\\pip-build-t8iw_px1\\robot
framework-autoitlibrary\\setup.py';f=getattr(tokenize, 'open', open)(__file__);c
ode=f.read().replace('\r\n', '\n');f.close();exec(compile(code, __file__, 'exec'
))" install --record C:\Users\BA05476\AppData\Local\Temp\pip-mb2nou3f-record\ins
tall-record.txt --single-version-externally-managed --compile" failed with error
 code 1 in C:\Users\BA05476\AppData\Local\Temp\pip-build-t8iw_px1\robotframework
-autoitlibrary\
You are using pip version 9.0.1, however version 19.1.1 is available.
You should consider upgrading via the 'python -m pip install --upgrade pip' comm
and.

C:\Temp\robotframework-autoitlibrary-1.2.4>

**Comment by acn-nithya** [comment]
This issue got resolved after changing the directory for the Python. I observed that the space in Program Files is the main issue when it came to the installation of setup.py  of AutoIT as while registering the DLL of this library the space in it threw the issue as the complete path was not passed. The directory which doesn't have any space doesn't really have any issue as observed.Guess this helps others as well facing the same issue as me. Thanks! 

++ Nithya
